src/lib.rs

```diff
@@ -27,6 +28,10 @@ pub struct Cli {
     /// Turns verbose console debug output on
     #[clap(short, long)]
     pub debug: bool,
+
+    // Make int number of concurrent requests
+    #[clap(short, long, default_value_t = 1)]
```

Comment:
Is there an upper limit to how many concurrent requests we want to allow for? Perhaps adding CLI interface data check on that would be in order. For example, do we really want to allow for 65k simultaneous threads, or at least without having tested it well? :)